src/pool/srv_pool.c

```diff
@@ -922,6 +924,91 @@ pool_svc_alloc_cb(d_iov_t *id, struct ds_rsvc **rsvc)
 	return rc;
 }
 
+static void
+pool_svc_get(struct pool_svc *svc)
+{
+	ds_rsvc_get(&svc->ps_rsvc);
+}
+
+static void
+pool_svc_put(struct pool_svc *svc)
+{
+	ds_rsvc_put(&svc->ps_rsvc);
+}
```

Comment:
can't we just call ds_rsvc_get/put() directly? what's the point of these functions?